app/controllers/posts_controller.rb

```diff
@@ -1,11 +1,61 @@
 class PostsController < ApplicationController
   def index
     @user = User.find(params[:user_id])
-    @posts = @user.posts
+    @posts = Post.all
   end
 
   def show
     @user = User.find(params[:user_id])
-    @post = @user.posts.find(params[:id])
+    @post = Post.find(params[:id])
+    @comment = Comment.new
+  end
+
+  def new
+    @current_user = User.first
+    @post = Post.new
+  end
+
+  def create
+    @current_user = User.first
```

Comment:
- [ ] There is no need to declare `@current_user` here again since it we already have the `#current_user` method in  `application_controller` that returns the first user in the db.